source/index.txt

```diff
@@ -163,9 +163,9 @@ Explore MongoDB Realm Services
 Build and Deploy an App
 -----------------------
 
-You can use any of the following interfaces to work with Realm application services:
+You can use any of the following interfaces to work with {+service-short+} application services:
 
-- :ref:`Realm UI <deploy-ui>`: Create and maintain your Realm App from a web browser
-- :ref:`Realm CLI <deploy-cli>`: Define and deploy your app with local configuration files
+- :ref:`{+ui+} <deploy-ui>`: Create and maintain your {+app+} from a web browser
```

Comment:
Hmm. This one is odd. I'm not sure renaming the source constant from "Realm app" to "Atlas App Service" really works, in terms of parsing the text. "Create and maintain your Atlas App Service from a web browser" seems weird, especially as this is about deploying config changes to the app. Is "Atlas App" a thing in the new naming? Or "Atlas App Services App"? I think one of those might be a better fit for the new value of this source constant.

Reply:
Product guidance on this point has been conflicting